### PR TITLE
Add aria-labelby support on graphs

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard14**
### Working

Give the option to pass a description or an ariaDescription property to a graph. 
It will link up the description or the hidden ariaDescription with the SVG by using the `aria-labelledby` by creating an unique ID to link them.

I've only added in the localize keys that didn't have a description shown right now.

### Could be improved
Right now I'm getting the data from the locale by using:
`const accessibilityGraphs = siteText.accessibility_grafieken as any;`
I've tried some things but couldn't get around some typescript warnings, and had to fix it with `as any` for now.

### CODE REVIEW

File: packages/app/src/locale/nl.json [hunk 1]
```diff
@@ -1931,5 +1931,39 @@
...
+      "verpleeghuiszorg_besmette_locaties": "Dit is een grafiek die in de verpleeghuiszorg besmette locaties over tijd weergeeft.",
+      "verpleeghuiszorg_overleden": "Dit is een grafiek die in de verpleeghuiszorg overledenen over tijd weergeeft.",
+      "gehandicaptenzorg_positief_getest": "Dit is een grafiek die in de gehandicaptenzorg positief getesten over tijd weergeeft.",
+      "gehandicaptenzorg_besmette_locaties": "Dit is een grafiek die in de gehandicaptenzorg besmette locaties over tijd weergeeft.",
+      "gehandicaptenzorg__overleden": "Dit is een grafiek die in de gehandicaptenzorg overledenen over tijd weergeeft.",
+      "thuiswonende_ouderen_besmettingen": "Dit is een grafiek die bij de thuiswonende ouderen besmettingen over tijd weergeeft.",
+      "thuiswonende_ouderen_overleden": "Dit is een grafiek die bij de thuiswonende ouderen overledenen over tijd weergeeft.",
+      "rioolwater_virusdeeltjes": "Dit is een grafiek die in het rioolwater virusdeeltjes over tijd weergeeft.",
+      "rioolwater_meetwaarde": "Dit is een grafiek die in het rioolwater de meetwaarde over tijd weergeeft."
+    },
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/locale/nl.json:1963]: I see no good reason to scope these things per nl/vr/gm really. They seem to be generic descriptions that seem to be constant across the charts in different scopes

**Review by VWSCoronaDashboard8 — CHANGES_REQUESTED** [review]

File: packages/app/src/locale/nl.json (hunk 1 above)
Comment (VWSCoronaDashboard14) [reply packages/app/src/locale/nl.json:1963]: This was specifically asked to order them like this.

**Review by VWSCoronaDashboard14 — COMMENTED** [review]
Removed unused useRef commit should be "Introduced uniqueID hook, removed any type, added assert, updated const names to be more declarative".

File: packages/app/src/locale/nl.json (hunk 1 above)
Comment (VWSCoronaDashboard8) [reply packages/app/src/locale/nl.json]: @VWSCoronaDashboard14 asked by who?

File: packages/app/src/components/choropleth/choropleth.tsx [hunk 2]
```diff
@@ -117,7 +118,7 @@ const ChoroplethMap: <T1, T3>(
     hoverRef,
   } = props;
 
-  const clipPathId = useRef(`_${Math.random().toString(36).substring(2, 15)}`);
+  const clipPathId = useRef(uniqueId());
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/components/choropleth/choropleth.tsx:121]: The useRef can be removed here.

File: packages/app/src/pages/gemeente/[code]/rioolwater.tsx [hunk 3]
```diff
@@ -31,7 +31,7 @@ import {
 } from '~/utils/sewer-water/municipality-sewer-water.util';
 
 const text = siteText.gemeente_rioolwater_metingen;
-const accessibilityGraphs = siteText.accessibility_grafieken as any;
+const accessibilityGraphsDescriptions = siteText.accessibility.grafieken;
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/pages/gemeente/[code]/rioolwater.tsx:34]: nit; `graphDescriptions` or even `descriptions` as a variable name would have been clear enough within this scope I think

File: packages/app/src/utils/useUniqueId.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import { useMemo } from 'react';
+
+// Create a unique identifier
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/utils/useUniqueId.ts:3]: nit; I think the comment doesn't add much value. The code and hook name speak for themselves I think.

**Review by VWSCoronaDashboard8 — APPROVED** [review]

**Review by VWSCoronaDashboard1 — APPROVED** [review]

**Review by VWSCoronaDashboard4 — APPROVED** [review]

**Pull request merged**